### PR TITLE
Cleanup of sources schema

### PR DESCRIPTION
**Pull request by thomasonw**
Removed redundent enteries, reloated N2K specific informaiton to N2K
section, scrubbed type of productId from string to number.

Referance RFC 0001

### CODE REVIEW

File: schemas/groups/sources.json [hunk 1]
```diff
@@ -34,15 +34,19 @@
...
                       "type": "object",
-                      "description": "NMEA 2000 pgn numbere",
+                      "description": "NMEA 2000 pgn number",
                       "patternProperties": {
                         "[0-9]*": {
                           "$ref": "#/definitions/timestamp"
                         }
                       }
                     },
+                    "manufactureId": {
```
Comment (timmathews) [inline schemas/groups/sources.json]: I think this should be `manufacturerId`

File: test/data/sources.json [hunk 2]
```diff
@@ -8,19 +8,17 @@
           "pgns": {
             "130312": ""
           },
-          "uniqueId": "1722191",
+          "uniqueId": 1722191,
+          "manufactureId": 437,
```
Comment (timmathews) [inline test/data/sources.json]: `manufacturerId`?

File: test/data/sources.json [hunk 3]
```diff
@@ -8,19 +8,17 @@
...
-          "name": "Digital Yacht",
-          "productId": "1739",
-          "modelId": "N2NET NMEA 2000 AIS",
-          "softwareVersion": "040200.08.03.00",
-          "hardwareVersion": "0",
-          "serialNumber": "0000000"
+          "deviceFunction": 140,
+          "productID": 1739
         },
+        "manufactureName": "Digital Yacht",
```
Comment (timmathews) [inline test/data/sources.json]: `manufacturerName`?

File: schemas/groups/sources.json (hunk 1 above)
Comment (thomasonw) [inline schemas/groups/sources.json]: Completed

File: test/data/sources.json (hunk 2 above)
Comment (thomasonw) [inline test/data/sources.json]: Completed

File: test/data/sources.json (hunk 3 above)
Comment (thomasonw) [inline test/data/sources.json]: Completed - I wonder why this one got past the npm test ???

**Pull request merged**